create_graph.py

```diff
@@ -16,6 +16,9 @@ def process_pkl_files(triples_file,labels_file):
 
 #Creates igraph object and a list of nodes
 def create_igraph_graph(edgelist_df,labels):
+    
+    #To ensure using column names is case insensitive
+    edgelist_df.columns = edgelist_df.columns.str.lower()
```

Comment:
will this cause an error in the next line which reorders the df?

Reply:
It didn't for me, it's just changing the columns names from Subject --> subject for example. Unless I am missing something?

Reply:
I got this error while running it: "KeyError: "None of [Index(['Subject', 'Object', 'Predicate'], dtype='object')] are in the [columns]"

Reply:
I can't reproduce this. I wonder if it was how I was passing the input files but I might just make the changes for the other issues I have and see if this is resolved

Reply:
ok. sounds good.